ui/src/components/member/Member.js

```diff
@@ -79,7 +79,7 @@ export default class Member extends React.Component {
         }
 
         let fullName = '';
-        let shortId = this.props.item.memberName.startsWith('user.')
+        let shortId = this.props.item.memberName.startsWith(`${process.env.NEXT_PUBLIC_USER_DOMAIN}`)
             ? this.props.item.memberName.substr(
                   5,
```

Comment:
`5` will need to be updated to length of new constant + 1.

Reply:
I'm trying to fix it like this, but It appears that `shortId` is not being used, so can I remove the logic regarding `shortId`?

```js
        const userDomain = `${process.env.NEXT_PUBLIC_USER_DOMAIN}.`;
        let shortId = this.props.item.memberName.startsWith(userDomain)
            ? this.props.item.memberName.substr(
                  userDomain.length,
                  this.props.item.memberName.length
              )
            : this.props.item.memberName;
```

Reply:
yes, it can be removed.

On another note, this file also needs to be updated: https://github.com/AthenZ/athenz/blob/06d9820526480c4887c7e40c541ab2cd581f51dc/ui/src/components/domain/CreateDomain.js#L149

Reply:
Thank you, I fixed it.